roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1187,11 +1189,12 @@ def get_defaults(self, roles):
                         iptables_sync_period='5s', set_node_ip=False)
             defaults['node'] = node
 
-        if 'nfs' in roles:
-            nfs = dict(exports_dir='/var/export', registry_volume='regvol',
-                       export_options='*(rw,sync,all_squash)')
-            defaults['nfs'] = nfs
-
+        defaults['hosted'] = dict(registry_storage_type=None,
+                                  registry_storage_host=None,
+                                  registry_storage_volume_dir='/var/exports',
+                                  registry_storage_volume='registry',
+                                  registry_storage_volume_capacity='5Gi',
+                                  registry_storage_export_options='*(rw,sync,all_squash)')
```

Comment:
Most of our variables are in the form `openshift_$ROLE_$VARIABLE` and this would make them be `openshift_$ROLE_$COMPONENT_$VARIABLE`. I'm okay with it but it restricts what can be passed through `oo_openshift_env`